### PR TITLE
test for running lots of DKGs back to back

### PR DESCRIPTION
**Pull request by CluEleSsUK**

### CODE REVIEW

File: demo/demo_test.go [hunk 1]
```diff
@@ -93,3 +94,40 @@ func testLocalOrchestration(t *testing.T) {
...
+
+func TestRunShitloadsOfDKGs(t *testing.T) {
+	sch, err := crypto.GetSchemeFromEnv()
+	require.NoError(t, err)
+	beaconID := test.GetBeaconIDFromEnv()
+
+	c := cfg.Config{
+		N:            3,
+		Thr:          2,
+		Period:       "4s",
```
Comment (AnomalRoil) [inline demo/demo_test.go:106]: having a shorter period, e.g. `1s`,  would make each DKG run faster, no?

File: demo/lib/orchestrator.go [hunk 2]
```diff
@@ -496,29 +500,34 @@ func (e *Orchestrator) RunResharing(resharingGroup *ResharingGroup, timeout time
...
 		}
 		fmt.Printf("\t- Joined DKG for node %s\n", n.PrivateAddr())
 	}
 
+	done := make(map[string]bool)
 	for _, n := range except(e.reshareNodes[1:], e.newNodes) {
 		n := n
+		if n.Index() == leader.Index() || done[n.PublicAddr()] {
+			continue
+		}
```
Comment (AnomalRoil) [inline demo/lib/orchestrator.go:513]: Can `done[n.PublicAddr()]` be true at any time in that loop? 😮 
Are there duplicated nodes in `except(e.reshareNodes[1:], e.newNodes)`?

**Review by AnomalRoil — APPROVED** [review]
Just tests, lgtm

File: demo/lib/orchestrator.go (hunk 2 above)
Comment (CluEleSsUK) [reply demo/lib/orchestrator.go:513]: for some reason this was failing - I didn't think there could be duplicate nodes, but this solved it

**Pull request merged**